### PR TITLE
tests:Added verify otp test cases

### LINKED ISSUE #40: Test Verify OTP
**Is your feature request related to a problem? Please describe.**
Add test cases for the Verify OTP API, namely:

**Describe the solution you'd like**
Relevant tests must be added to `tests/users/tests_verify_otp_api.py`

### PR DESCRIPTION
**Pull request by sahil9001**
### Description
Added test cases for verify OTP 

**Type of Change:** (Delete irrelevant options)
- Code
- Testing

**Code/Quality Assurance Only** (Delete irrelevant options)

- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Ran `python manage.py test` and passed locally

```
Logs:

Creating test database for alias 'default'...
System check identified no issues (0 silenced).
...............
===================== Mock email service =====================

Recipients: ['user234@example.com']
Subject: E-Cell NITRR
Body: Your OTP for E-Cell NIT Raipur portal is 4518.

==============================================================

......
===================== Mock email service =====================

Recipients: ['user234@example.com']
Subject: E-Cell NITRR
Body: Your OTP for E-Cell NIT Raipur portal is 2691.

==============================================================

...
===================== Mock email service =====================

Recipients: ['crash.test.dummy@gmail.com']
Subject: E-Cell NITRR
Body: Your OTP for E-Cell NIT Raipur portal is 1896.

==============================================================

..........
----------------------------------------------------------------------
Ran 34 tests in 2.411s

OK
Destroying test database for alias 'default'...

```

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged
- [ ] Update Swagger documentation and the exported file at /docs folder
- [ ] Update requirements.txt

**Code/Quality Assurance**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Review by priyanshisharma — APPROVED** [review]
Looks good to me 🚀 ✨

File: tests/users/tests_verify_otp_api.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        self.assertEqual(response.status_code,status.HTTP_200_OK)
+
+    #Test fail with already verified otp
+    def test_fail_already_verified(self):
+        auth_client = self.create_auth_client()
+        data = {
+            "otp" : self.user.otp
+        }
+        response = auth_client.post(self.verify_otp_api,data)
+        response = auth_client.post(self.verify_otp_api,data)
```
Comment (anshsrtv) [inline tests/users/tests_verify_otp_api.py:77]:
```suggestion
        # Trying to Verify again
        response = auth_client.post(self.verify_otp_api,data)
```

**Review by anshsrtv — COMMENTED** [review]
Nice work again 🙌  Will be waiting for [AuthAPITestCase](https://github.com/ECellNitrr/teesco-backend/blob/dev/tests/AuthAPITestCase.py) since that can be used here as well 😄

**Comment by sahil9001** [comment]
Done!

File: tests/users/tests_verify_otp_api.py (hunk 1 above)
Comment (sahil9001) [reply tests/users/tests_verify_otp_api.py:56]: Done

**Review by anshsrtv — APPROVED** [review]

**Pull request merged**